packages/api/src/routes/sdkRoutes.ts

```diff
@@ -61,6 +61,51 @@ export const sdkRoutesWrapper = (storeManager: Promise<dbManager.DataStoreManage
         }
     });
 
+    sdkRoutes.get("/api/sdk/rulesbyflag", auth.isAuthenticated, async (req: Request, res: Response) => {
+        try { 
+            const flagKey = req.query.flagKey as string;
+            const flag = await flagsService.getRulesByFlag(flagKey);
+            res.json({
+                user: req.user,
+                rulesByFlag: flag
+            });
+        } catch (error) {
+            console.log(error);
+            res.status(500).json({
+                error: SdkResponseCodes.GenericError
+            });
+        }
+    });
 
+    sdkRoutes.get("/api/sdk/segments", auth.isAuthenticated, async (req: Request, res: Response) => {
+        try { 
+            const segments = await segmentsService.getSegments("");
+            res.json({
+                user: req.user,
+                segments: segments
+            });
+        } catch (error) {
+            console.log(error);
+            res.status(500).json({
+                error: SdkResponseCodes.GenericError
+            });
+        }
+    });
+
+    sdkRoutes.get("/api/sdk/conditionsbysegment", auth.isAuthenticated, async (req: Request, res: Response) => {
```

Comment:
to keep consitency you can call the url: `/segments/:segmentKey/conditions` and read the data from the `req.params`